amr-wind/equation_systems/PDE.H

```diff
@@ -71,6 +71,9 @@ public:
             m_fields, m_sim.has_overset(), variable_density,
             m_sim.has_mesh_mapping()));
         m_src_op.init_source_terms(m_sim);
+
+        // clip (if needed) and fillpatch
```

Comment:
I would remove this comment, since a lot of different stuff can happen in the post_solve_op